docs/development/deploying-locally/index.md

```diff
@@ -1,3 +1,45 @@
 # Deploying locally
 
 REANA can be easily deployed locally on your laptop using Minikube.  Useful for REANA platform development.
+
+Clone the main REANA repository:
+
+```console
+$ git clone https://github.com/reanahub/reana.git && cd reana
+```
+
+You can setup a REANA development as follows:
+```console
+$ GITHUB_USER=<YOUR-GITHUB-USERNAME> make setup clone prefetch
```

Comment:
- This would be wrong syntax, so better use something like `${YOUR_GITHUB_USERNAME}` in examples
- We can standardise on `YOUR_...`  as leading text to find all places to customise, including PVC, SSO....
- Or let's simply use "johndoe" everywhere as a persona, with `reana-johndoe.cern.ch` that we already have